screens/HomeScreen.js

```diff
@@ -95,12 +149,12 @@ export default class HomeScreen extends React.Component {
                 styles.codeHighlightContainer,
                 styles.homeScreenFilename
               ]}>
-              <MonoText>{this.state.points} / 1000 </MonoText>
+              <MonoText>{this.state.points} total points </MonoText>
             </View>
             <Text style={styles.getStartedText}>
               {' '}
               {`${1000 -
-                parseInt(this.state.points)} points to your next reward`}
+                parseInt(this.state.points) % 1000} points to your next reward`}
```

Comment:
I added a column to Airtable called 'Unlocked Rewards' in the Customers table which does the calculation via formula!

Reply:
I think it's separate from this but I'll do some more handling in the next PR!